data/tools.yml

```diff
@@ -27,6 +27,7 @@
   homepage: "https://afterthedeadline.com/"
```

Comment:
I assume the `data/tools.yml` should be removed when you're done, right? Or do we keep it *and* have the individual files for each tool? 🤔

Reply:
We should remove it once we are happy with the final structure. The same applies to the converter scripts.